src/Stryker.CLI/Stryker.CLI.UnitTest/Stryker.CLI.UnitTest.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Perhaps while we're at it we should move the version to Directory.build.props so we have one consistent version number for all projects.